src/identity/index.ts

```diff
@@ -5,6 +5,7 @@ export { getAvatar } from './core/getAvatar';
 export { getName } from './core/getName';
 export { useAvatar } from './hooks/useAvatar';
 export { useName } from './hooks/useName';
+export { IdentityProvider } from './hooks/useIdentity';
```

Comment:
Ok love this, althought part of me thinks it should be more explicit, maybe `OnchainKitIdentityProvider`, but I think folks can probably customize that..... let's see for now is fine.

Reply:
or, if all OnchainKit components will have some shared `config`, should we instead do `OnchainKitProvider`?

What do you think @kyhyco @cnasc ?

Reply:
`OnchainKitProvider` sounds good to me. Let me tinker with it.